pkgs/applications/editors/focuswriter/default.nix

```diff
@@ -1,24 +1,37 @@
-{ lib, fetchurl, pkg-config, qmake, qttools, hunspell, qtbase, qtmultimedia, mkDerivation }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, hunspell
+, qtbase
+, qtmultimedia
+, qttools
+, qt5compat
+}:
 
-mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "focuswriter";
-  version = "1.7.6";
+  version = "1.8.5";
 
-  src = fetchurl {
-    url = "https://gottcode.org/focuswriter/focuswriter-${version}-src.tar.bz2";
-    sha256 = "0h85f6cs9zbxv118mjfxqfv41j19zkx2xq36mpnlmrlzkjj7dx9l";
+  src = fetchFromGitHub {
+    owner = "gottcode";
+    repo = "focuswriter";
+    rev = "v${version}";
+    hash = "sha256-6wvTlC/NCCcN2jpwqtoOsCln3ViY/vj7NpMsbYHBGiI=";
   };
 
-  nativeBuildInputs = [ pkg-config qmake qttools ];
-  buildInputs = [ hunspell qtbase qtmultimedia ];
+  nativeBuildInputs = [ pkg-config cmake qttools ];
+  buildInputs = [ hunspell qtbase qtmultimedia qt5compat ];
+
+  dontWrapQtApps = true;
```

Comment:
Why aren't we wrapping it?

Reply:
I'd had issues early on when attempting to wrap it however it looks like those are gone now.

fixed in https://github.com/NixOS/nixpkgs/pull/253111/commits/1f95a84bdaff12b56bcb653c9251dba6206c8f70